sql/mozfun/glean/legacy_compatible_experiments/metadata.yaml

```
@@ -0,0 +1,2 @@
+description: Formats a Glean experiments field into a Legacy Telemetry experiments field by dropping the extra information that Glean collects
```

```suggestion
description: >
  Formats a Glean experiments field into a Legacy Telemetry experiments field
  by dropping the extra information that Glean collects
```
Oh, for yaml files the max line width is 80 characters, so we need to split this across multiple lines